### PR TITLE
add clusters page

### PR DESCRIPTION
**Pull request by akeller**
## What is the purpose of the change

Adding more info and visibility into our development clusters for Professional plan users.

## Are there related marketing activities

No.

## When should this change go live?

Development clusters are already available, but users may not know they exist! As soon as this is out of draft state and reviewed, it should be published.

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by akeller** [comment]
Changes have only been applied to `/next/` but should also be applied to current docs prior to merging.

**This PR adds a new page!**

There is a TODO item for a definition of a cluster. Also, we need to clarify if trial clusters are different and should also be defined on this page.

**Comment by akeller** [comment]
Internal collab doc - https://docs.google.com/document/d/1tN3mlc2rjr4UhH5w-vlItLQ-ZMhyfUiH7ZCPDI8sUX0/edit?usp=sharing

**Comment by akeller** [comment]
@christinaausley I incorporated some of Jakob's content from the Google Doc linked above. Can you take a look at this page? I'm particularly interested in your thoughts on flow and if we should include additional content from the Google Doc, which is why I added you as an assignee and not a reviewer. (I was a little worried some of the Google Doc was too marketing oriented.)

File: docs/components/concepts/clusters.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## Production clusters
+
+Production clusters come in three sizes: small (S), medium (M), and large (L). To learn more about the size of cluster best suited for your use case, see our [Best Practices](/components/best-practices/best-practices-overview.md) for more information on [sizing your runtime environment](/components/best-practices/architecture/sizing-your-environment.md#sizing-your-runtime-environment).
+
+When you deploy and execute your [BPMN](/components/modeler/bpmn/bpmn.md) or [DMN](/components/modeler/dmn/dmn.md) models on a production cluster, this might impact your monthly (Professional) or annual (Enterprise) total fee, meaning the more you execute your models, the higher your total fee may be.
+
+### Free Trial cluster
+
+A Free Trial cluster is a special kind of production cluster. Free Trial clusters are size small, and only available during your trial period.
```
Comment (christinaausley) [inline docs/components/concepts/clusters.md:28]: "A Free Trial cluster is a special kind of production cluster." -- This is a bit vague. What exactly do we mean by this? Unless we want to specify further, I would likely remove this sentence.

File: docs/components/concepts/clusters.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Once you sign up for a Free Trial, you are able to spin up one production cluster for the time of your trial.
```
Comment (christinaausley) [inline docs/components/concepts/clusters.md:30]: I noticed we use "spin up" or "spun up" several times in this doc. Is this how we want to refer to cluster creation, or should we just say "you are able to create one production cluster," for example?

File: docs/components/concepts/clusters.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+|              | Development | Production - S | Production - M | Production - L |
+| ------------ | ----------- | -------------- | -------------- | -------------- |
+| Free         | \-          | \-             | \-             | \-             |
+| Free Trial   | \-          | X              | \-             | \-             |
+| Professional | X           | X              | \-             | \-             |
+| Enterprise   | X           | X              | X              | X              |
+
+## Production clusters
+
+Production clusters come in three sizes: small (S), medium (M), and large (L). To learn more about the size of cluster best suited for your use case, see our [Best Practices](/components/best-practices/best-practices-overview.md) for more information on [sizing your runtime environment](/components/best-practices/architecture/sizing-your-environment.md#sizing-your-runtime-environment).
```
Comment (christinaausley) [inline docs/components/concepts/clusters.md:22]: "Production clusters come in three sizes: small (S), medium (M), and large (L)." Could we note this before we reference the table above since it contains information required to understand the table (S, M, L)?

File: docs/components/concepts/clusters.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Development clusters
+
+Development clusters are recommended for development, testing, proof of concepts, and demos.
+
+Professional Plan users have the option to create **development clusters**, offering free execution for development. Deployment and execution of models (process instances, decision instances, and task users) is provided at no cost.
+
+Additionally, the following applies to **development clusters**:
+
+- **Cluster is not High-Available & Less Hardware**: Reduced hardware resources and availability compared to production cluster (for example, one Zeebe node only). No 99.5% SLA, but still sufficient for most development use cases.
```
Comment (christinaausley) [inline docs/components/concepts/clusters.md:42]: Is there a particular reason why "High-Available & Less Hardware" is uppercase?

"No 99.5% SLA, but still sufficient for most development use cases." -- This is also a bit vague. Can you help explain what you mean by this, or use cases where this may *not* be sufficient?

File: docs/components/concepts/clusters.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+- **Shorter history of processes and decisions**: Data retention in Operate, Optimize, and Tasklist is reduced to one day. For example, pending or historical process instances are deleted after one day.
+- **Cluster auto-pause**: We will pause your cluster if it's not used for two hours. You can wake up your cluster by clicking a button in Console. This means not all functionality will work all the time. For example, BPMN timers and BPMN message catch events will not work while clusters are paused.
```
Comment (christinaausley) [inline docs/components/concepts/clusters.md:44]: "We will pause your cluster if it's not used for two hours." -- To clarify, I would maybe reword this as something like "Your cluster is paused if it goes unused for two hours" or something of the like.

"You can wake up your cluster by clicking a button in Console." -- Do you mean you can click any button in Console to get your cluster working again?

File: docs/components/concepts/clusters.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+When you deploy and execute your [BPMN](/components/modeler/bpmn/bpmn.md) or [DMN](/components/modeler/dmn/dmn.md) models on a production cluster, this might impact your monthly (Professional) or annual (Enterprise) total fee, meaning the more you execute your models, the higher your total fee may be.
+
+### Free Trial cluster
+
+A Free Trial cluster is a special kind of production cluster. Free Trial clusters are size small, and only available during your trial period.
+
+Once you sign up for a Free Trial, you are able to spin up one production cluster for the time of your trial.
+
+When your trial expires, you automatically move to the Free Plan. This plan allows you to model BPMN and DMN collaboratively, but does not support execution of your models. Any cluster spun up during your free trial is deleted, and you cannot spin up new clusters.
```
Comment (christinaausley) [inline docs/components/concepts/clusters.md:32]: "When your trial expires, you automatically move to the Free Plan." -- I'm trying to think of a way to clarify this. For example, "When your Free Trial plan expires, you are automatically transferred to the Free Plan."

File: docs/components/concepts/clusters.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+A cluster is a provided group of nodes that run Camunda Platform 8. By default, Camunda Platform 8 clusters are production-ready.
+
+Professional and Enterprise plan customers can create as many production or development clusters as they want.
+
+The following table shows each plan and available type or size of cluster:
+
+|              | Development | Production - S | Production - M | Production - L |
+| ------------ | ----------- | -------------- | -------------- | -------------- |
+| Free         | \-          | \-             | \-             | \-             |
```
Comment (christinaausley) [inline docs/components/concepts/clusters.md:15]: Should Free Trial technically come before the Free Plan?

**Review by christinaausley — COMMENTED** [review]
A general technical review in my latest commit for stylistic changes, and some comments for conversation.

I tried to adjust some wording so this isn't too "market-y" and think the way it is written is concise, clear, and to the point for interested users.

File: docs/components/concepts/clusters.md (hunk 1 above)
Comment (akeller) [reply docs/components/concepts/clusters.md:28]: We can remove this sentence or combine it with the next sentence. Which do you prefer?

Somewhat rephrasing but extra context - Free Trial clusters have all the features/functionality of a production cluster, but are size small and only available during your trial period. You cannot convert a Free Trial cluster to a different kind of cluster.

File: docs/components/concepts/clusters.md (hunk 2 above)
Comment (akeller) [reply docs/components/concepts/clusters.md:30]: This is a good call out. The "spin up" or "spun up" terminology implies that creation is fast and easy, but in the end, it's really just cluster creation. Could you clean this up for us?

File: docs/components/concepts/clusters.md (hunk 4 above)
Comment (akeller) [reply docs/components/concepts/clusters.md:42]: C&P keywords from marketing. I'll fix.

Re: SLA, let's see if we can find a page on the website about our SLAs if people want to click through for clarification.

File: docs/components/concepts/clusters.md (hunk 5 above)
Comment (akeller) [reply docs/components/concepts/clusters.md:44]: > "You can wake up your cluster by clicking a button in Console." -- Do you mean you can click any button in Console to get your cluster working again?

This is question back to Jakob or Felix. I'll follow up in the working doc.

File: docs/components/concepts/clusters.md (hunk 6 above)
Comment (akeller) [reply docs/components/concepts/clusters.md:32]: I like this because it emphasizes the plan moving.

File: docs/components/concepts/clusters.md (hunk 7 above)
Comment (akeller) [reply docs/components/concepts/clusters.md:15]: I like this. Like the order you could experience the plans.

File: docs/components/concepts/clusters.md (hunk 3 above)
Comment (akeller) [reply docs/components/concepts/clusters.md:22]: With this change, I deleted the Production cluster section.

File: docs/components/concepts/clusters.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+Development clusters are recommended for development, testing, proof of concepts, and demos.
+
+Professional Plan users have the option to create **development clusters**, offering free execution for development. Deployment and execution of models (process instances, decision instances, and task users) is provided at no cost.
+
+Additionally, the following applies to **development clusters**:
+
+- **Cluster is not high-available & less hardware**: Reduced hardware resources and availability compared to production cluster (for example, one Zeebe node only). No 99.5% SLA, but still sufficient for most development use cases.
+- **Shorter history of processes and decisions**: Data retention in Operate, Optimize, and Tasklist is reduced to one day. For example, pending or historical process instances are deleted after one day.
+- **Cluster auto-pause**: Your cluster is paused if it goes unused for two hours. You can wake up your cluster by clicking a button in Console. This means not all functionality will work all the time. For example, BPMN timers and BPMN message catch events will not work while clusters are paused.
```
Comment (felix-mueller) [inline docs/components/concepts/clusters.md:42]: Auto-Pause is not implemented yet

File: docs/components/concepts/clusters.md (hunk 5 above)
Comment (akeller) [reply docs/components/concepts/clusters.md]: This is a future item, we'll need to remove it from the PR.

File: docs/components/concepts/clusters.md (hunk 8 above)
Comment (akeller) [reply docs/components/concepts/clusters.md:42]: I'll remove it!

File: docs/components/concepts/clusters.md (hunk 2 above)
Comment (christinaausley) [reply docs/components/concepts/clusters.md:28]: Adjusting in latest commit 👍

File: docs/components/concepts/clusters.md (hunk 4 above)
Comment (christinaausley) [reply docs/components/concepts/clusters.md]: https://camunda.com/legal/terms/camunda-platform/camunda-platform-8-saas/ could work

**Review by christinaausley — APPROVED** [review]

File: docs/components/concepts/clusters.md (hunk 1 above)
Comment (akeller) [reply docs/components/concepts/clusters.md:26]: Adding in commit where I'll also backport for 8.1

File: docs/components/concepts/clusters.md (hunk 4 above)
Comment (akeller) [reply docs/components/concepts/clusters.md]: Following up on Slack with this resource. I noticed a discrepancy in how we talk about SLAs and Support SLAs. I'm going to move forward with this PR and we can always update later.

**Review by christinaausley — APPROVED** [review]
